### DISCUSSION TITLE
Nitrogen cant access libgtkmm

### DISCUSSION THREAD

**Discussion by justbenuk** [i3/i3#4952] [Q&A]
I am unable to open nitrogen because of the following error
`
nitrogen: error while loading shared libraries: libgtkmm-2.4.so.1: cannot open shared object file: No such file or directory
`
I have tried to google for more information but couldn't find anything. This is a fresh install of manjaro i3 community edition. 

I have tried to reinstall nitrogen - as well as the Libbtkmm file but nothing appears to resolve it.

**Comment by Airblader** [comment]
gtkmm doesn't seem to be installed correctly, or not in the correct version. I think you need to install extra/gtkmm.

> as well as the Libbtkmm file

You shouldn't copy individual files into your system, it'll rarely work and just give you more problems in the future.
